apps/drec-ui/src/shared/modals/userModals/index.ts

```diff
@@ -0,0 +1 @@
+export * from './UserModalsCenter';
```

Comment:
Storing User Modals components in `shared` is not a good idea.
By `shared` previously we meant only shared-state (like storing token, etc.).
Those modals should be a part of `components` folder